lib/index.js

```diff
@@ -11,6 +11,18 @@ import KEYS from "./visitor-keys.js";
 // List to ignore keys.
 const KEY_BLACKLIST = new Set([
     "parent",
+
+    // "type",
+
+    // `BaseNodeWithoutComments`
+    // "range",
+
+    // `SourceLocation`
+    // "loc",
+
+    // `Comment`
+    // "comments",
+    // "innerComments",
```

Comment:
Can we remove these lines? We usually avoid having commented-out code in the codebase unless it is associated with a TODO action.

Reply:
Sure. I had thought these were going to be removed in a future version, so should I add a comment to do so? File an issue? Is there any particular approach for planning ahead with to-dos to occur before breaking changes?

`eslint-plugin-unicorn` has a truly excellent `expiring-todo-comments` rule which can be tied to version or dependency changes which will cause linting to fail unless applied. I might recommend unicorn for `eslint-config-eslint` anyways.

Reply:
I think it would be best to discuss this in an issue first.

Reply:
Ok, thanks for the feedback. Added #38 and removed the comments.

And FWIW, re: Unicorn, I filed https://github.com/eslint/eslint/pull/15731 . Even when not heavily using to-dos in code, having those conditional to-dos can really help ensure that version-dependent to-dos don't get lost.